README.md

```diff
@@ -850,26 +850,27 @@ You can call it multiple times for different layers.
 Example: `set_global_routing_layer_pitch Metal6 1.34`.
 
 ```
-set_clock_routing [-clock_pdrev_fanout fanout] \
-                  [-clock_topology_priority priority]
+set_clock_routing [-pdrev_fanout fanout] \
+                  [-pdrev_alpha alpha]
 ```
 The `set_clock_routing` command sets specific configurations for clock nets.
 Options description:
-- **clock_pdrev_fanout**: Set the minimum fanout to use PDRev for the routing topology construction of the clock nets (e.g.: -clock_pdrev_fanout 5)
-- **clock_topology_priority**: Set the PDRev routing topology construction priority for clock nets.
-See `set_pdrev_topology_priority` command description for more details about PDRev and topology priority (e.g.: -topology_priority 0.6)
+- **pdrev_fanout**: Set the minimum fanout to use PDRev for the routing topology construction of the clock nets (e.g.: -pdrev_fanout 5)
+- **pdrev_alpha**: Set the PDRev routing topology construction trade-off for clock nets. See `set_pdrev_alpha` command description for
+more details about PDRev and topology trade-off.
+Example: `set_clock_routing -pdrev_fanout 5 -pdrev_alpha 0.6`
 
 ```
-set_pdrev_topology_priority netName alpha
+set_pdrev_alpha [-net net_name] alpha
 ```
 FastRoute has an alternative tool for the routing topology construction, called PDRev. You can define the topology construction
```

Comment:
This needs work. It assumes the user knows the fastroute routing topology.
Fastroute itself is completely hidden from the user to even using its name will lead to confusion.
fastroute? global route? huh?

By default the global router uses steiner trees to construct route guides. A steiner tree minimizes the total wire length.